### PR TITLE
LIT 4991 6251

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
I made CAe 4991 to a general record for lists of the months, of which we do have some. To sort them out in versions is a task that cannot be done right now.  https://github.com/BetaMasaheft/Documentation/issues/2244

### CODE REVIEW

File: 4001-5000/LIT4991Months.xml [hunk 1]
```diff
@@ -45,12 +55,30 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                    <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BLadd16211"/>
+                    <relation name="lawd:hasAttestation" active="LIT4991Months" passive="EMIP00636"/>
+                    <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BDLaeth34"/>
+                    <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BLadd24990"/>
+                    <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BMLor148"/>
+                      <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BLorient766"/>
+                      <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BAVet40"/>
+                        <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BAVet24"/>
+                        <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BAVet15"/>
+                    <relation name="dcterms:isPartOf" active="LIT4991Months" passive="LIT2293Sawase"/>
```
Comment (eu-genia) [inline 4001-5000/LIT4991Months.xml:79]: add different from LIT6251Months relation

File: 4001-5000/LIT4991Months.xml [hunk 2]
```diff
@@ -45,12 +55,30 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                    <relation name="lawd:hasAttestation" active="LIT4991Months" passive="IVorlov20"/>
+                    <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BLadd16211"/>
+                    <relation name="lawd:hasAttestation" active="LIT4991Months" passive="EMIP00636"/>
+                    <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BDLaeth34"/>
+                    <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BLadd24990"/>
+                    <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BMLor148"/>
+                      <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BLorient766"/>
+                      <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BAVet40"/>
+                        <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BAVet24"/>
+                        <relation name="lawd:hasAttestation" active="LIT4991Months" passive="BAVet15"/>
```
Comment (eu-genia) [inline 4001-5000/LIT4991Months.xml:78]: what is with bnfabb39? this or another new one?

Comment (eu-genia) [reply 4001-5000/LIT4991Months.xml:78]: (bnfabb39  has to be updated)

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: 4001-5000/LIT4991Months.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply 4001-5000/LIT4991Months.xml:78]: Yes, added. Abb 39 will be updated.

**Pull request merged**